.github/workflows/compile.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     - name: Init opam
       run: opam init -y
     - name: Install sail
-      run: opam pin -y sail 0.16
+      run: opam pin -y sail 0.17.1
```

Comment:
If this actually works, please can you do a PR to bump the devcontainer build to use it as well?

Possibly in a follow-on PR, it would be nice to have a nightly / weekly / something CI job that tested with the latest (unpinned) opam sail so we can catch incompatibilities early.

Reply:
OK, I've created https://github.com/CHERIoT-Platform/devcontainer/pull/3